File: 2026-03/src/day_10/boj_11694/Main.java
package day_10.boj_11694;

import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;

        int nimsum = 0;
        int cnt = 0;

        int n = Integer.parseInt(br.readLine());
        st = new StringTokenizer(br.readLine());
        for (int i = 0; i < n; i++) {
            int p = Integer.parseInt(st.nextToken());
            nimsum ^= p;
            if (p == 1) cnt++;
        }

        if ((cnt < n && nimsum != 0) || (cnt == n && cnt % 2 == 0)) {
            System.out.println("koosaga");
        } else {
            System.out.println("cubelover");
        }
    }
}

File: 2026-03/src/day_10/boj_11694/Main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int nimsum = 0;
    int cnt = 0;

    int n;
    cin >> n;

    for (int i = 0; i < n; i++) {
        int p;
        cin >> p;
        nimsum ^= p;
        if (p == 1) cnt++;
    }

    if ((cnt < n && nimsum != 0) || (cnt == n && cnt % 2 == 0)) {
        cout << "koosaga";
    } else {
        cout << "cubelover";
    }
}
